apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -68,7 +68,10 @@ public SpannerCommand()
         {
             SpannerConnection = connection;
             _transaction = transaction;
-            Parameters = parameters;
+            if (parameters != null)
+            {
+                Parameters = parameters;
+            }
```

Comment:
Instead of initializing in two spots and adding an if, why not just null coalesce? 

`Parameters = parameters ?? new SpannerParameterCollection();`

That will keep the compile error if you create a new constructor that forgets to initialize.